<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QWidget" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>367</width>
         <height>372</height>
        </rect>
       </property>
       <widget class="QTextBrowser" name="chatTextBrowser">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>361</width>
          <height>371</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="msgLineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="sendPushButton">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>0</y>
     <width>431</width>
     <height>170</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="usernameLabel">
        <property name="text">
         <string>Username:</string>
        </property>
        <property name="buddy">
         <cstring>usernameLineEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="usernameLineEdit">
        <property name="text">
         <string>suqdqi</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loginPushButton">
        <property name="text">
         <string>Login</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="logoutPushButton">
        <property name="text">
         <string>Logout</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="serverPortLabel">
          <property name="text">
           <string>Server Port: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="serverPortLineEdit">
          <property name="text">
           <string>5001</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="serverIPLabel">
          <property name="text">
           <string>Server IP:   </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="serverIPLineEdit">
          <property name="text">
           <string>127.0.0.1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QComboBox" name="interfacesComboBox"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="localPortLabel">
          <property name="text">
           <string>My Port:       </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="localPortLineEdit">
          <property name="text">
           <string>3001</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="localIPLabel">
          <property name="text">
           <string>My IP:       </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="localIPLineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="remotePortLabel">
          <property name="text">
           <string>Reciver Port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="remotePortLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="remoteIPLabel">
          <property name="text">
           <string>Reciver IP:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="remoteIPLineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>43</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="onlineUsersListWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>194</y>
     <width>191</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="onlineUsersLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>170</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Online Users:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendFile_button">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>selec a file</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="errorEdit">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>270</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>30</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>270</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>error precent</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>330</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>corrupted packet recieved</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>360</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>packet sent</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>390</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>packet recived </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>420</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>retransmitted packtes</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorRec">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>330</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="recv">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>390</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="sent">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>360</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="resend">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>420</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>440</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
